projects/observability/src/shared/components/explore-query-editor/series/explore-query-series-group-editor.component.test.ts

```diff
@@ -127,6 +127,6 @@ describe('Explore Query Series Group Editor component', () => {
     spectator.queryAll(ExploreQuerySeriesEditorComponent)[0].seriesChange.emit(buildSeries('modified first'));
 
     expect(onChange).toHaveBeenCalledTimes(1);
-    expect(onChange).toHaveBeenCalledWith([matchSeriesWithName('modified first'), matchSeriesWithName('second')]);
+    expect(onChange).toHaveBeenCalledWith([matchSeriesWithName('modified_first'), matchSeriesWithName('second')]);
```

Comment:
this one's actually interesting. I believe we use name and alias as the output of that regex, and maybe it would be better to leave name untouched and just do the replace on the alias? I guess can go with this since it matches existing behavior for now (and I don't know if code relies on the name in other ways), and revisit if needed. I'm guessing the effect is that something like this would show up in the legend with underscores rather than spaces.

Reply:
I would prefer to do it later if rise some issue